tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,22 +6,7 @@
    "source": [
```

Comment:
<p>Thanks for adding this, I think it helps for adding I think this is another diversion. Maybe this can be come a how-to guide. I feel this entire section should be a separate document, maybe a Explanation? We haven't talked much about explanation but it seems to fit there. I have encountered many users asking about the qiskit bit order (little endian). I think it would be good to have a single place to refer to about this ordering, why it's chosen and how to convert from little endian to big endian.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>